docker-compose.yml

```
@@ -0,0 +1,45 @@
+    volumes:
+      - ./backend:/app
+      - tunetrace-data:/app/data
+    networks:
+      - tunetrace-network
+
+volumes:
+  tunetrace-data:
+
+networks:
+  tunetrace-network:
+    driver: bridge 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)